DXRMissions/DeusEx/Classes/DXRMissionsM05.uc

```diff
@@ -293,3 +293,13 @@ function AfterMoveGoalToLocation(Goal g, GoalLocation Loc)
         }
     }
 }
+
+function MissionTimer()
+{
+    switch(dxr.localURL) {
+    case "05_NYC_UNATCOMJ12LAB":
+        UpdateGoalWithRandoInfo('FindPaul', "Paul could be located anywhere in the lab.");
```

Comment:
we might want to mention that you can see Paul with the security camera

Reply:
Added a mention of the camera in 179cefd